File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Policy/PolicyStatusChangeAudit.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

using System.Collections.Generic;

namespace RMA.Service.ClientCare.Contracts.Entities.Policy
{
    public class PolicyStatusChangeAudit
    {
        public int PolicyStatusChangeAuditId { get; set; }
        public int PolicyId { get; set; }
        public PolicyStatusEnum PolicyStatus { get; set; }
        public string Reason { get; set; }
        public System.DateTime EffectiveFrom { get; set; }
        public System.DateTime? EffectiveTo { get; set; }
        public string RequestedBy { get; set; }
        public System.DateTime RequestedDate { get; set; }

        public Policy Policy { get; set; }
        public List<Policy> VapsPolicies { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/policy-status-change-audit.ts
import { PolicyStatusEnum } from '../enums/policy-status.enum';
import { Policy } from './policy';

export class PolicyStatusChangeAudit {
  policyStatusChangeAuditId: number;
  policyId: number;
  policyStatus: PolicyStatusEnum;
  reason: string;
  effectiveFrom: Date;
  effectiveTo: Date;
  requestedBy: string;
  requestedDate: Date;

  policy: Policy;
  vapsPolicies: Policy[];
}
